src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -573,6 +578,11 @@ public override SyntaxNode CreateFieldDeclaration(IFieldSymbol field, CodeGenera
                 }
             }
 
+            if (options.Options == null)
```

Comment:
`options.Options ??=`

Reply:
If I'm not mistaken, since options.Options doesn't have a set method, I don't think we can assign to it. We could have options.Options support a set method, but I don't know if this is the right move since none of the existing settings in CodeGenerationOptions have set methods.

Reply:
I see